pkgs/applications/networking/mailreaders/claws-mail/default.nix

```diff
@@ -41,18 +41,19 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
-  patches = [ ./mime.patch ];
+  patches = [ ./mime.patch ./disable-python-detection.patch ];
 
   postPatch = ''
     substituteInPlace src/procmime.c \
         --subst-var-by MIMEROOTDIR ${shared-mime-info}/share
   '';
 
-  nativeBuildInputs = [ pkgconfig wrapGAppsHook ];
+  nativeBuildInputs = [ autoreconfHook pkgconfig wrapGAppsHook python2Packages.wrapPython ];
+  propagatedBuildInputs = with python2Packages; [ python ] ++ optional enablePluginPython [ pygtk pygobject2 ];
```

Comment:
You should use `optionals` with lists.

Reply:
and done, I think. Maybe I misunderstood you.